labs/ipfs/reference-content/install-ipfs-desktop.mdx

```diff
@@ -0,0 +1,78 @@
+---
+meta:
+  title: Using Scaleway IPFS Pinning with IPFS Desktop
+  description: This page shows you how use Scaleway IPFS Pinning with IPFS Desktop
+content:
+  h1: Using Scaleway IPFS Pinning with IPFS Desktop
+  paragraph: This page shows you how to use Scaleway IPFS Pinning with IPFS Desktop
+tags: ipfs pinning storage ipfs-storage volume desktop labs web3
+dates:
+  validation: 2023-09-26
+  posted: 2023-09-26
+categories:
+  - labs
+---
+
+With Scaleway IPFS pinning, you can permanently store your data on the public IPFS network. You can either [create](/labs/ipfs/api-cli/volumes-operations) and manage your volume  via the [Kubo client](https://docs.ipfs.tech/install/command-line/) or through [IPFS Desktop](https://docs.ipfs.tech/install/ipfs-desktop/). 
+
+This article shows you how to add Scaleway as your remote pinning service using IPFS Desktop.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+    - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)
+    - You have generated your [API key](/identity-and-access-management/iam/how-to/create-api-keys/)
+    - You have downloaded and installed [IPFS Desktop](https://docs.ipfs.tech/install/ipfs-desktop/)
+    - You have created a [volume](/labs/ipfs/how-to/create-volume/)
+</Message>
+
+## Retrieve the ID of your volume
+
+1. Click **IPFS Pinning** in the **Labs** section of the [Scaleway console](https://console.scaleway.com) side menu. The IPFS Pinning section displays.
+2. Click the <Icon name="seeMore" /> icon next to your desired volume. The **Pins** section displays.
+3. Click the **Settings** tab next to **Pins**. The **Settings** section displays.
+4. Retrieve the ID of your volume. You will need it for the next procedure.
+
+
+## Add Scaleway as a remote pinning service
+
+This operation allows you to add Scaleway as a remote service in your IPFS desktop.
+
+1. Open IPFS Desktop.
+2. Click **Settings** on the sidebar menu. The settings panel displays.
+3. Click the **Add Service** button on the **Pinning services** section of the panel. A pop-up displays asking you to select a pinning service provider.
+4. Click **Add a custom one** to add Scaleway as a remote pinning service. A configuration form displays.
+5. Fill in the different fields with the following information:
+    - Nickname: `Scaleway`
+    - API endpoint: `https:/<your_volume_region_ID>.ipfs.labs.scw.cloud/<your_volume_ID>`
+    - Secret access token: `<SCW_secret_key>`
+  <Message type="important">
+    Don't forget to replace the parameters in brackets with the relevant information.
+  </Message>
+
+  <Message type="note">
+    Different endpoints are available depending on the regions you created your volume in.
```

Comment:
Just read this after I left my previous comment. I think it would be neater to explain the parameter (fr-par; nl-ams, pl_waw) rather than the whole endpoint.